packages/vitest/src/node/pools/threads.ts

```diff
@@ -1,5 +1,5 @@
 import { MessageChannel } from 'node:worker_threads'
-import { cpus } from 'node:os'
+import { availableParallelism, cpus } from 'node:os'
```

Comment:
This will just fail because `node:os` doesn't export this method on lower Node.js versions. And you cannot access undeclared named exports.

Using default export might be better

Reply:
done, or do you have another suggestion?